### PR TITLE
fix: add datatable footer

### PR DESCRIPTION
**Pull request by seungyeoneeee**
### To Reviewers
- [ ] Self-reviewed (coding conventions, bug-free, functionality verified, tests checked, documentation updated)
- [ ] Minor change, review optional (`style`, `chore`, `ci`, straightforward changes, etc.)
- [ ] Previously reviewed in feature branch, no further review needed
- [x] Need review or discussion

### Description (optional)
Add DataTable Footer (about each column summary)

### CODE REVIEW

File: packages/mirinae/src/data-display/tables/data-table/PDataTable.vue [hunk 1]
```diff
@@ -358,32 +455,43 @@ export default defineComponent<DataTableProps>({
...
+            proxySortDesc: useProxyValue<boolean | undefined>(
+                'sortDesc',
+                props,
+                emit,
+            ),
         });
 
+        const slots = useSlots();
+        const isExpandable = (_slots: SetupContext['slots']) => _slots['tf-col-format'] !== undefined || _slots.foot !== undefined;
+
```
Comment (WANZARGEN) [inline packages/mirinae/src/data-display/tables/data-table/PDataTable.vue:467]: nitpick: I think using `computed` api would be better!

File: packages/mirinae/src/data-display/tables/data-table/PDataTable.vue [hunk 2]
```diff
@@ -319,20 +399,33 @@ export default defineComponent<DataTableProps>({
             type: Boolean,
             default: false,
         },
+        footerTitle: {
+            type: String,
+            default: 'Total',
+        },
```
Comment (WANZARGEN) [inline packages/mirinae/src/data-display/tables/data-table/PDataTable.vue:405]: issue: please remove unnecessary prop

File: apps/web/src/services/advanced/components/WorkspaceGroupTabWorkspace.vue [hunk 3]
```diff
@@ -183,6 +183,14 @@ watch(() => workspaceGroupPageGetters.selectedWorkspaceGroupId, () => {
...
     await workspaceGroupPageStore.fetchCostReportConfig();
 });
+
+const reduce = (arr: (number | undefined)[] | any) => arr.reduce((acc, value) => acc + (value ?? 0), 0);
+
+const costInfoReduce = (arr: (number | {month: any})[] | any) => {
+    const result = arr.reduce((acc, cur) => (acc + Object.keys(cur).includes('month') ? cur?.month : 0), 0);
+
+    return result;
+};
```
Comment (WANZARGEN) [inline apps/web/src/services/advanced/components/WorkspaceGroupTabWorkspace.vue:193]: issue: I think 'month' is the specific context. Please check it.

**Review by WANZARGEN — CHANGES_REQUESTED** [review]
Thanks!

File: apps/web/src/services/advanced/components/WorkspaceManagementTable.vue [hunk 4]
```diff
@@ -274,7 +282,7 @@ onMounted(async () => {
                 <div class="th-tooltip">
                     <span>{{ field.label }}</span>
                     <p-tooltip
-                        :contents="$t('IAM.WORKSPACES.TOOLTIP_STATE')"
+                        :contents="String($t('IAM.WORKSPACES.TOOLTIP_STATE'))"
```
Comment (piggggggggy) [inline apps/web/src/services/advanced/components/WorkspaceManagementTable.vue:285]: 😯

File: apps/web/src/services/asset-inventory/helpers/asset-analysis-data-table-helper.ts [hunk 5]
```diff
@@ -65,6 +65,7 @@ export const getRefinedAssetAnalysisTableData = (results: MetricDataAnalyzeResul
         _results
             .filter((d) => d.date === realtimeDate)
             .forEach((d) => {
+                console.log('dsjfl;sdkjf', d);
```
Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/helpers/asset-analysis-data-table-helper.ts:68]: nitpick: `console.log` !

**Review by piggggggggy — APPROVED** [review]
LGTM~ I left small comment

**Review by WANZARGEN — APPROVED** [review]

**Pull request merged**